mandatory/2-function-creation.js

```diff
@@ -25,7 +35,15 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+  typeof arrayOfStrings === "array";
+    let myArray = [];
+    for (var i = 0; i < arrayOfStrings.length; i++) {
```

Comment:
Should use `let` instead of `var`, also inside for loops.

Reply:
Noted, thank you.